test/open_mic_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+require './lib/open_mic'
+
+class OpenMicTest < Minitest::Test
+  def setup
+    @joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    @joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+    @jokes = [@joke_1, @joke_2]
+    @user_1 = User.new("Name")
+    @user_2 = User.new("Name2")
+    @open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+  end
+
+  def test_it_exists
+    assert_instance_of OpenMic, @open_mic
+  end
+
+  def test_it_has_attributes
+    assert_equal "Comedy Works", @open_mic.location
+    assert_equal "11-20-18", @open_mic.date
+    assert_equal [], @open_mic.performers
+  end
+
+  def test_for_welcome
+    @open_mic.welcome(@user_1)
+    assert_equal @user_1, @open_mic.performers[0]
```

Comment:
like the `jokes` test for the user, it would be good to assert_equal against a whole array, not just one element, like `assert_equal [@user_1], @open_mic.performers` so we know that's ALL that's in the array